configure

```diff
@@ -9,6 +9,7 @@ SOURCE_BASE_DIR=`pwd -P`
 popd > /dev/null
 
 # This file contains customized config settings.
+rm -f .bazelrc
```

Comment:
Otherwise, when you rerun ./configure, .bazelrc file will still contain the content produced by your last run, possibly with different env value.